packages/core/src/schema/models/apiTokens.ts

```diff
@@ -0,0 +1,22 @@
+import { InferSelectModel } from 'drizzle-orm'
+import { bigserial, text, timestamp, uuid, varchar } from 'drizzle-orm/pg-core'
+
+import { latitudeSchema } from '..'
+import { timestamps } from '../schemaHelpers'
+import { workspaces } from './workspaces'
+
+export const apiTokens = latitudeSchema.table('api_tokens', {
+  id: bigserial('id', { mode: 'bigint' }).notNull().primaryKey(),
+  token: uuid('token')
+    .notNull()
+    .unique()
+    .$defaultFn(() => crypto.randomUUID()),
```

Comment:
All logs are going to be associated with an `apiToken` (for traceability), so we might want index performance on the `id`.

Reply:
The token is to authorized the api calls. Also the logs would be on Elastic don't add any index here I think